diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+_Graces-MacBook-Pro:~ grace$~/wdi/try$_
```

Comment:
Technically correct but remember that when you do 'pwd' to print the working directory, it will have '/Users/grace/wdi/try' instead of 'Graces-MacBook-Pro'